src/components/Button.jsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Button = ({ label, onClick, isDisabled = false, className }) => {
```

Comment:
Try to write optional props at last
`const Button = ({ label, onClick, className, isDisabled = false }) => {`